bin/sidekiq

```diff
@@ -1,6 +1,7 @@
 #!/bin/bash
 if [ ${RAILS_ENV:-development} != "production" ]; then
   bundle check || bundle
+  yarn
```

Comment:
I'm not sure what we're trying to check, but you may want `yarn install --check-files` instead.